pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "cim-graph"
-version = "v0.1.230404"
+version = "0.1.0c0"
```

Comment:
c?

Reply:
No idea how that happened. We shouldn't be touching this manually. weird github actions thing?